tools/benchmark/etc/trafficserver/header_rewrite.config

```diff
@@ -0,0 +1,5 @@
+cond %{READ_REQUEST_PRE_REMAP_HOOK} [AND]
```

Comment:
I can't find anywhere these URLs would be used. Neither can I find where `header_rewrite` would be used either.